ddtrace/contrib/pyramid/__init__.py

```diff
@@ -14,6 +14,11 @@
 
     # use your config as normal.
     config.add_route('index', '/')
+
+If you use the 'pyramid.tweens' settings value to set the tweens for your
+application, you need to add 'ddtrace.contrib.pyramid:trace_tween_factory'
+explicitely to the list.
```

Comment:
explicitly

Reply:
Other than that, can we add a snippet to show `you need to add 'ddtrace.contrib.pyramid:trace_tween_factory' explicitly to the list`? It's always better to write a small sentence + code so they can get immediately what they have to do without reading anything else.